nahusenayElias/Team_4
# DRD-64: Create Project Case content type + content

## Related ticket / Customer approval:
[DRD-64](https://edu-team4-react24k.atlassian.net/browse/DRD-64?atlOrigin=eyJpIjoiMDA4ZGRhOWRmM2NjNGQzNTgxYmVlODk5ODMyOTBlOTgiLCJwIjoiaiJ9)
## In this PR:
- Created content type project_case
- Created text, image, text with image paragraph types
## Testing instructions:
- Checkout branch
- Import drupal db (`lando db-import DRD-64.gz`)
- `lando drush cim`
- Check you see content "Project A", "Project B" and "Project C" in drupal


[DRD-64]: https://edu-team4-react24k.atlassian.net/browse/DRD-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
